components/Rating/rating.html

```diff
@@ -0,0 +1,110 @@
+    <!-- main body area -->
+        <main class="main">
+            <section class="intro">
+                <h1 class="para-heading" >RATING</h1>
+                <p class="doc-para">
+                    Rating component is used mainly to give a feedback on a product or service.  
+                </p>                
+
+                <!-- STAR RATING -->
+                <h2 class="para-heading" id="s-rating">Star Rating</h2>
+                <p class="doc-para">
+                    Given below is a simple star rating component , which on click get filled up to give a final rating response.
+                </p>
+                <div class="comp-container">
+                    <div class="rating-container">
+                        <input type="radio" name="rate" id="rate-a">
+                        <label for="rate-a"></label>
+                        <input type="radio" name="rate" id="rate-b">
+                        <label for="rate-b"></label>
+                        <input type="radio" name="rate" id="rate-c">
+                        <label for="rate-c"></label>
+                        <input type="radio" name="rate" id="rate-d">
+                        <label for="rate-d"></label>
+                        <input type="radio" name="rate" id="rate-e">
+                        <label for="rate-e"></label>
+                    </div>  
+                </div> 
+                
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28200%2C178%2C200%2C1%29&t=seti&wt=none&l=htmlmixed&width=746.25&ds=true&dsyoff=15px&dsblur=68px&wc=true&wa=true&pv=46px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522rating-container%2522%253E%250A%2520%2520%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520name%253D%2522rate%2522%2520id%253D%2522rate-a%2522%253E%250A%2520%2520%2520%2520%253Clabel%2520for%253D%2522rate-a%2522%253E%253C%252Flabel%253E%250A%2520%2520%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520name%253D%2522rate%2522%2520id%253D%2522rate-b%2522%253E%250A%2520%2520%2520%2520%253Clabel%2520for%253D%2522rate-b%2522%253E%253C%252Flabel%253E%250A%2520%2520%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520name%253D%2522rate%2522%2520id%253D%2522rate-c%2522%253E%250A%2520%2520%2520%2520%253Clabel%2520for%253D%2522rate-c%2522%253E%253C%252Flabel%253E%250A%2520%2520%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520name%253D%2522rate%2522%2520id%253D%2522rate-d%2522%253E%250A%2520%2520%2520%2520%253Clabel%2520for%253D%2522rate-d%2522%253E%253C%252Flabel%253E%250A%2520%2520%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520name%253D%2522rate%2522%2520id%253D%2522rate-e%2522%253E%250A%2520%2520%2520%2520%253Clabel%2520for%253D%2522rate-e%2522%253E%253C%252Flabel%253E%250A%253C%252Fdiv%253E%2520%2520"
+                style="width: 100%; height: 374px; border:0; transform: scale(1); overflow:hidden; margin-bottom: 3rem; margin-top: 1rem;"
```

Comment:
inline css can be moved to a CSS file for consistency. It is not breaking anything.

Reply:
yes, will try to change it later on.